src/test/java/tests/SortingByPriceTest.java

```diff
@@ -0,0 +1,63 @@
+package tests;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import pages.HomePage;
+import pages.WomenPage;
+
+import java.text.ParseException;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static util.Waiters.WAITE_10;
+import static util.Waiters.waitImplicit;
+
+/**
+ * Created by : Volodymyr_Silitskyi
+ * Created at : 8/16/2018
+ */
+
+
+public class SortingByPriceTest {
+    WebDriver driver = new ChromeDriver();
```

Comment:
Driver initialization better to move in separate class like `WebDriverManager` and create there static method `getDriver`. The idea is create flexibility in instantiation WebDriver for test. For example, you want to run your test also in IE and Firefox. In your case you will create `if else` in this test class. Then you will need do this for other tests. So you will need to duplicate your code. Imagine you have 100 tests.

Good approach is to move driver managing responsibility to separate class, also you can use java properties to setup configuration, like create property `webdriver=Chrome` that determine browser you want to run. In your manager class you need read that property and create needed instance of WebDriver.

Also, `getDriver` method can be setup differently. If you want to have one browser at any time - it should be singleton. If you want to have new instance each time you call `getDriver` it should be factory.

Everything that I described here is part of Test Automation Framework design. I know it is hard to put all things together but I decided to share this info right here :) This will make your life easier in future.